apps/docs/content/components/Hero/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Hero
+description: Use the hero component to render a full-width banner on a webpage
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image displaying a hero with a heading, description, primary and secondary actions.](https://github.com/primer/brand/assets/912236/51f3dd23-9ac7-4825-9169-4fb7f295e7f1)
+
+- Heading: the main title of the hero.
+- Description: a short description of the hero.
+- Actions: the main and secondary actions the user should take on the page.
+
+## Usage
+
+Use the hero component to provide a title, description and actions to introduce the main content of a page. The hero component is designed to be positioned at the top of a page and spans the full width of the viewport.
```

Comment:
Total nit.. a grammatical suggestion to use a relative clause that reduce repetition of 'to'...

```suggestion
Use the hero component to provide a title, description and action that introduces the main content of a page. The hero component is designed to be positioned at the top of a page and spans the full width of the viewport.
```